src/D2L.Bmx/Okta/Models/MfaType.cs

```diff
@@ -0,0 +1,13 @@
+namespace D2L.Bmx.Okta.Models;
+
+internal record MfaOption(
+	string Name,
+	MfaType Type
+);
+
+// TODO: More robust / generic way of handling MFA Types
+public enum MfaType {
```

Comment:
this can be internal as well
```suggestion
public internal MfaType {
```